server/routes/index.test.ts

```diff
@@ -39,124 +43,173 @@ afterEach(() => {
   jest.resetAllMocks()
 })
 
-describe('GET /', () => {
-  it('should render index page', () => {
-    return request(app)
-      .get('/')
-      .expect('Content-Type', /html/)
-      .expect(res => {
-        expect(res.text).toContain('Equiniti Historical Data')
-      })
+describe('routes', () => {
```

Comment:
Just added a describe to surround all the tests, as it makes it easier to run all tests in the file in the IDE.